backend/Application/Services/UserService.cs

```diff
@@ -78,6 +83,65 @@ public async Task<Response> ChangePasswordAsync(ChangePasswordRequest requestMod
         return new Response(true, "Success");
     }
 
+    public async Task<Response<CreateUserResponse>> CreateUserAsync(CreateUserRequest requestModel)
+    {
+        var userRepository = UnitOfWork.AsyncRepository<User>();
+
+        var user = new User();
+
+        var responseModel = new CreateUserResponse(user);
+
+        if (GetAge(requestModel.DateOfBirth) < 18)
+        {
+            return new Response<CreateUserResponse>(false, ErrorMessages.InvalidAge, responseModel);
+        }
+
+        if (DateTime.Compare(requestModel.DateOfBirth, requestModel.JoinedDate) != 1
+            || requestModel.JoinedDate.DayOfWeek == DayOfWeek.Saturday || requestModel.JoinedDate.DayOfWeek == DayOfWeek.Sunday)
```

Comment:
Break this check to short line
DateTime.Compare(requestModel.DateOfBirth, requestModel.JoinedDate) != 1
            || requestModel.JoinedDate.DayOfWeek == DayOfWeek.Saturday 
            || requestModel.JoinedDate.DayOfWeek == DayOfWeek.Sunday